cmd/vcfFilter/vcfFilter.go

```diff
@@ -235,6 +244,8 @@ func main() {
 	var segregatingSitesOnly *bool = flag.Bool("segregatingSitesOnly", false, "Retains only variants that are segregating in at least one sample.")
 	var removeNoAncestor *bool = flag.Bool("removeNoAncestor", false, "Retains only variants with an ancestor allele annotated in the info column.")
 	var onlyPolarizableAncestors *bool = flag.Bool("onlyPolarizableAncestors", false, "Retains only variants that can be used to construct a derived allele frequency spectrum. Must have a subsitution where the ancestral allele matches either alt or ref.")
+	var weakToStrongOnly *bool = flag.Bool("weakToStrongOnly", false, "Retains only variants that are weak to strong mutations.")
+	var noWeakToStrong *bool = flag.Bool("noWeakToStrong", false, "Removes weak to strong variants.")
```

Comment:
These flag descriptions are pretty ambiguous. More explanation would be helpful (even just a G/C <-> A/T)